packages/ui/src/components/app-sidebar/sidebar-user.tsx

```diff
@@ -12,12 +12,7 @@ const UserAvatar = ({ user }: Pick<UserProps, 'user'>) => {
 
   return (
     <>
-      <Avatar.Root className="mr-2 size-8 rounded-lg">
-        {!!user?.url && <Avatar.Image src={user.url} alt="user" />}
-        <Avatar.Fallback className="rounded-lg bg-sidebar-background-5 text-sidebar-foreground-1">
-          {getInitials(userName)}
-        </Avatar.Fallback>
-      </Avatar.Root>
+      <Avatar name={userName} src={user?.url} size="default" rounded className="mr-2" />
```

Comment:
We can drop the `size` here since it's using "default", which is, umm, the default.
```suggestion
      <Avatar name={userName} src={user?.url} rounded className="mr-2" />
```